troubleshooting.md

```diff
@@ -0,0 +1,550 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+
```

Comment:
I'm using VSCode toc plugin, if you're using something similar, adding specific headers will update toc and links when there's any update in the document

Reply:
In my case, after changing the other titles it would require before:

`<!-- TOC depthFrom:2 insertAnchor:false orderedList:true updateOnSave:true withLinks:true -->`

and 

`<!-- /TOC -->`

 after the index